README.md

```diff
@@ -0,0 +1,372 @@
++-------------------------------------------------------------------------------+
+||                                   Regions                                   ||
+|+-----------------------------------+-----------------------+-----------------+|
+||             Endpoint              |      OptInStatus      |   RegionName    ||
+|+-----------------------------------+-----------------------+-----------------+|
+||  ec2.eu-north-1.amazonaws.com     |  opt-in-not-required  |  eu-north-1     ||
+||  ec2.ap-south-1.amazonaws.com     |  opt-in-not-required  |  ap-south-1     ||
+||  ec2.eu-west-3.amazonaws.com      |  opt-in-not-required  |  eu-west-3      ||
+||  ec2.eu-west-2.amazonaws.com      |  opt-in-not-required  |  eu-west-2      ||
+||  ec2.eu-west-1.amazonaws.com      |  opt-in-not-required  |  eu-west-1      ||
+||  ec2.ap-northeast-2.amazonaws.com |  opt-in-not-required  |  ap-northeast-2 ||
+||  ec2.ap-northeast-1.amazonaws.com |  opt-in-not-required  |  ap-northeast-1 ||
+||  ec2.sa-east-1.amazonaws.com      |  opt-in-not-required  |  sa-east-1      ||
+||  ec2.ca-central-1.amazonaws.com   |  opt-in-not-required  |  ca-central-1   ||
+||  ec2.ap-southeast-1.amazonaws.com |  opt-in-not-required  |  ap-southeast-1 ||
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+
+```
+
+### Install moactl
+
+Download the [latest release of moactl](https://github.com/openshift/moactl/releases/latest) and add it to your path.
+
+Verify your installation by running the following command:
+
+```
+$ moactl
+Command line tool for MOA.
+
+Usage:
+  moactl [command]
+
+Available Commands:
+  completion  Generates bash completion scripts
+  create      Create a resource from stdin
+  delete      Delete a specific resource
+  describe    Show details of a specific resource
+  edit        Edit a specific resource
+  help        Help about any command
+  init        Applies templates to support Managed OpenShift on AWS clusters
+  list        List all resources of a specific type
+  login       Log in to your Red Hat account
+  logout      Log out
+  logs        Show logs of a specific resource
+  verify      Verify resources are configured correctly for cluster install
+  version     Prints the version of the tool
+
+Flags:
+      --debug     Enable debug mode.
+  -h, --help      help for moactl
+  -v, --v Level   log level for V logs
+
+Use "moactl [command] --help" for more information about a command.
+```
+
```

Comment:
add a blurb about using moactl bash completion here.

Reply:
I added some text below. This did not work for me on a Mac. We should add another oneliner for mac.